app/components/Editor.js

```diff
@@ -34,15 +34,7 @@ export default class Editor extends React.Component {
         for(let attribute of element.attributes) {
           newElement.setAttribute(attribute.nodeName, attribute.nodeValue);
         }
-
-        if(newElement.tagName != element.tagName) {
```

Comment:
@ningsongshen  Can you explain your reasoning as to why you think we don't need this anymore? I forget exactly even how to test this usecase, so I'm curious how you tested it..

Reply:
Oh sorry just noticed you mentioned this in your comments. Let me investigate..

Reply:
Hmm, it looks like resizing an image no longer persists its updated size. Even if you make another change to trigger a save, the image always retains its original size.

Reply:
It seems like we don't need it because all the information is contained in the `<img>` tag. From what I see, the `<figure>` tag is mainly to help Redactor identify it for image resizing and positioning purposes.

Reply:
Fixed in latest commit